packages/sdk/package.json

```diff
@@ -0,0 +1,114 @@
+{
+    "name": "@ripio/sdk",
+    "version": "1.1.0",
```

Comment:
the sdk package should increase to version 2.0, right?

Reply:
Yes, I'm trying to figure out if it can be done with semantic release